### PR TITLE
BETA: Register thebestdev.is-a.dev

### PR DESCRIPTION
**Pull request by notthebestdev**
Added `thebestdev.is-a.dev` using the [dashboard](https://manage.is-a.dev).

### CODE REVIEW

**Review by wdhdev — CHANGES_REQUESTED** [review]
This will need to be put on a nested subdomain `_discord.thebestdev.is-a.dev`. You will need to own `thebestdev.is-a.dev` first however.

**Comment by Omxpro** [comment]
you didnt understand what william said?

**Comment by notthebestdev** [comment]
> you didnt understand what william said?
> 
> 

what do you mean I understood what he said

**Comment by notthebestdev** [comment]
> > > you didnt understand what william said?
> 
> > 
> 
> > what do you mean I understood what he said
> 
> 
> 
> you didnt, you just made a normal domain with discord txt verification. you first have to own the domain and then you can make _discord.insertyourdomain.json for discord txt verification

the first time yes but I put an URL now

File: domains/thebestdev.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "notthebestdev",
+        "email": "thebestdeveloper@outlook.fr"
+    },
+    "record": {
+        "TXT": "dh=d04b3166526bba7742cb84fb32a18b421b85325a"
```
Comment (CuteDog5695) [inline domains/thebestdev.json:7]: then why is this .json file doing here?

Comment (notthebestdev) [reply domains/thebestdev.json:7]: idk I made it with the dashboard

Comment (CuteDog5695) [reply domains/thebestdev.json:7]: you'd have to delete it in your fork to remove it in this PR btw

**Pull request closed without merging**